src/js/other-websites/inpage_menu.js

```diff
@@ -60,13 +60,13 @@ async function getCachedServerStoragePref() {
       method: "getServerStoragePref",
     });
   } else {
-    return serverStoragePrefInStorage;
+    return serverStoragePref.server_storage;
```

Comment:
This fixes a bug where the inpage-menu was NOT correctly querying the `server_storage` pref.